runtimes/datascience/ubi9-python-3.11/utils/requirements-elyra.txt

```diff
@@ -1,24 +1,24 @@
 # Copied from: https://github.com/elyra-ai/elyra/blob/main/etc/generic/requirements-elyra.txt
 
 # This is a comprehensive list of python dependencies that Elyra requires to execute Jupyter notebooks.
-ipykernel==6.29.5
-ipython==8.27.0
-ipython-genutils==0.2.0
-jinja2==3.1.4
-jupyter-client==8.6.3
-jupyter-core==5.7.2
-MarkupSafe==2.1.5
-minio==7.2.8
-nbclient==0.10.0
-nbconvert==7.16.4
-nbformat==5.10.4
-papermill==2.6.0
-pyzmq==26.2.0
-prompt-toolkit==3.0.47
-requests==2.32.3
-tornado==6.4.1
-traitlets==5.14.3
-urllib3==2.2.3
```

Comment:
can you clean the entire requirements.txt and include these comments
```
# This requirements.txt file is for the exclusive reason to escape falling into https://github.com/opendatahub-io/elyra/blob/0ec8a571df6b3614aca91f20fe0b1dbdc8743dc6/elyra/pipeline/kfp/processor_kfp.py#L1099, 
incase the details are need please follow the comprehensive list of python dependencies present here:
https://raw.githubusercontent.com/opendatahub-io/elyra/refs/heads/main/etc/generic/requirements-elyra.txt
```
in all the places.

Reply:
interesting, I did not think that file was read anywhere but in runtime bootstrapper py